scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -60,23 +59,13 @@ check-try-runtime:
     - .compiler-info
   script:
     - |
-      export has_runtimemigration_label=$(curl -sS -H "Accept: application/vnd.github+json" \
-        -H "Authorization: token $GITHUB_PR_TOKEN" \
-        https://api.github.com/repos/paritytech/polkadot/issues/$CI_COMMIT_REF_NAME/labels | grep "E0-runtime_migration" | wc -l)
-    - |
-      if [[ $has_runtimemigration_label != 0 ]]; then
-        echo "Found label runtime_migration. Running tests"
-        export RUST_LOG=remote-ext=debug,runtime=debug
-        echo "---------- Running try-runtime for ${NETWORK} ----------"
-        time cargo build --locked --release -p "$NETWORK"-runtime
-        time cargo run --locked --release --features try-runtime try-runtime \
-            --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
-            -lruntime=debug \
-            --chain=${NETWORK}-dev \
-            on-runtime-upgrade live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
-      else
-        echo "runtime_migration label not found. Skipping"
-      fi
+      export RUST_LOG=remote-ext=debug,runtime=debug
+      echo "---------- Running try-runtime for ${NETWORK} ----------"
+      time cargo build --release --locked -p "$NETWORK"-runtime
+      time cargo run --locked --release --features try-runtime try-runtime \
+          --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
+          --chain=${NETWORK}-dev \
```

Comment:
Did you check the default for the `--checks` argument?  
May be worth to specify that explicitly so we know that the pre-and-post checks are actually running.

Reply:
Default is `pre-and-post`, even though the docs say otherwise. PR to update docs to match code: https://github.com/paritytech/substrate/pull/14467


I'll specify `--checks` here explicitly here anyway just so there's no ambiguity.